utils/uniset/src/uniset.rs

```diff
@@ -32,14 +32,15 @@ const ALL_SLICE: &ZeroSlice<u32> =
 /// Provides exposure to membership functions and constructors from serialized [`UnicodeSets`](UnicodeSet)
 /// and predefined ranges.
 #[derive(Debug, Eq, PartialEq, Clone, Yokeable, ZeroFrom)]
+#[cfg_attr(feature = "crabbake", derive(crabbake::Bakeable), crabbake(path = icu_uniset))]
 pub struct UnicodeSet<'data> {
     // If we wanted to use an array to keep the memory on the stack, there is an unsafe nightly feature
     // https://doc.rust-lang.org/nightly/core/array/trait.FixedSizeArray.html
     // Allows for traits of fixed size arrays
-
-    // Implements an [inversion list.](https://en.wikipedia.org/wiki/Inversion_list)
-    inv_list: ZeroVec<'data, u32>,
-    size: usize,
+    /// Implements an [inversion list.](https://en.wikipedia.org/wiki/Inversion_list)
+    pub inv_list: ZeroVec<'data, u32>,
+    /// The size of the set
+    pub size: usize,
```

Comment:
Issue: These fields have invariants, and there is a custom Deserialize impl to reflect that. I think there is already an unsafe constructor that you can use, but if not, you can make one

Reply:
Fixed by #1926